articles/relnotes/index.md

```diff
@@ -16,6 +16,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.x
+
+*Release date: October 30th, 2019*
+
+This release contains the following:
+
+- Updates to the Python installation, Python can now be installed using conda
+- Updates to Visual Studio Code & Visual Studio extensions to include language server support 
```

Comment:
Language server support has been there for a while, but with 0.10, the language server is now deployed as a self-contained executable such that the VS Code extension does not require the .NET Core SDK to be installed.